data/hai/hai reveal 5 philosophers.txt

```diff
@@ -151,16 +151,22 @@ mission "Hai Reveal [B02] Join the Fleet"
 		system "Alcyone"
 		fleet "Marauder fleet X" 4
 	# IX is 4 decent ships each, X is 6 ships but 2 are weak each. Total Fleet Size is roughly 28 + some chaff (~8).
-
+	on fail
+		dialog `You have failed an essential mission. To complete this story line, revert to the autosave or another earlier snapshot of the game.`
 	on accept
 		fail "Hai Reveal [B01] Information Blackout"
 		fail "Hai Reveal [A11-B] Permanent Fleets"
 	on enter "Sumar"
 		dialog `You have reached the rendezvous and various fleets are arriving. You have never seen so many military vessels in one place. As promised, both Hai and human ships have tied their navigation to your systems for coordinating the next couple of jumps to Alcyone.`
+	on enter "Moktar"
+		dialog `Some of your escorts are traveling from far away, and could run out of fuel before Stormhold. Make sure you refuel them before traveling onward.`
```

Comment:
If there's a problem with the escorts running out of fuel, then the variants used need more fuel. You are not meant to stop or land anywhere in this mission between departing Sunracer and landing on Stormhold.

Reply:
Then the escorts should start at your planet, rather than all over the galaxy.

Reply:
If the escorts start where you are, it fixes most of the problems in the mission sequence. I will make that change and retest.

Reply:
I see what you were trying to do with "escort waiting." The escorts don't wait. They'll fly to you, just like any other escorts, and run out of fuel. 

I'm reworking the mission to have the escorts start where you are (Sunracer), just like they do in all other missions.  Then the fuel supply won't matter.